### ISSUE TITLE
Accessibility: High Contrast dark theme

### ISSUE THREAD

**Issue by falcon-Z** [pop-os/pop#2825]
**Distribution (run `cat /etc/os-release`):**
 
 NAME="Pop!_OS"
VERSION="22.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 22.04 LTS"
VERSION_ID="22.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=jammy
UBUNTU_CODENAME=jammy
LOGO=distributor-logo-pop-os
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

Settings -> Accessibility -> High Contrast

**Issue/Bug Description:**

When enabling high contrast while using a dark theme the UI switches to a light theme (High contrast light theme) which really hurts the eye. As a visually impaired person some people could only read text over darker backgrounds such as a black background with while text. This is really important for Accessibility and i would like to request that the high contrast themes be interpreted based on the System theme or provide a drop down next to High contrast toggle allowing users to choose between light and dark themes respectively

**Steps to reproduce (if you know):**

**Expected behavior:**

When turning on High contrast the theme should be interpreted weather to be light or dark from System's theme

**Other Notes:**
